bpf/Makefile

```diff
@@ -49,6 +56,9 @@ c/fmt:
 .PHONY: format-check
 format-check:
 
+# TODO(kakkoyun): Fix.
+# clang-14: error: argument unused during compilation: '--gcc-toolchain=/nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0' [-Werror,-Wunused-command-line-argument]
+
```

Comment:
let's fix this first

Reply:
As I explained in other communication channels, this is a part of a series of PRs, which will be deprecated. This is an inherent issue because of the nix toolchain interfering with what we do. The clang picks up and environment variable and adds this implicit flag. I don't think it's productive to waster time on this at this stage, hence the TODO.